### ISSUE TITLE
Implement 

### ISSUE THREAD

**Issue by ayan-biswas0412** [ayan-biswas0412/AlgoDocs#18]

**Comment by susheelg1197** [comment]
I would like to perform binary search using golang

**Comment by ayan-biswas0412** [comment]
> I would like to perform binary search using golang

yeah go ahead

**Comment by suranaakshat02** [comment]
I would like to perform Binary Search using C

**Comment by ayan-biswas0412** [comment]
> I would like to perform Binary Search using C

Yeah sure go ahead, your contribution will help this project to grow more.

**Comment by Chaitanyaprasad60** [comment]
I would like to Perform binary Search in JavaScript.

**Comment by ayan-biswas0412** [comment]
> I would like to Perform binary Search in JavaScript.

Yeah sure go ahead, your contribution will help this project to grow more.

**Comment by sumitmohite0501** [comment]
I would like to do it in cpp

**Comment by ayan-biswas0412** [comment]
> I would like to do it in cpp

cpp is done I think

**Comment by Tanisha1721** [comment]
Could you please assign this issue to me ?  I would like to do this in java.
